src/css/styles.css

```diff
@@ -562,6 +570,15 @@ html {
   filter: var(--blue-filter); /* blue color */
 }
 
+/**
+ * TOOLTIP specific styling
+ *
+ * Should remove the numbering from the lists that come with the linguistic breakdowns
+ */
+.tooltip > span > ol {
```

Comment:
We're using BEM as a way to modularize our CSS. The reason to use BEM is that `span > ol` selectors are quite brittle (what happens if you remove the `<span>`?)

Here's a primer on how to create BEM classes: http://getbem.com/naming/

I think what would need to happen is that the `<ol>` would get a class like `.linguistic-breakdown` as its **BLOCK**, and these styles would apply to it.